tajo-core-tests/src/test/java/org/apache/tajo/engine/query/TestAlterTable.java

```diff
@@ -103,10 +107,33 @@ public final void testAlterTableAddPartition() throws Exception {
     assertTrue(fs.exists(partitionPath));
     assertTrue(partitionPath.toString().indexOf("col3=1/col4=2") > 0);
 
+    boolean existPartition = false;
+    List<CatalogProtos.TablePartitionProto> allPartitions = catalog.getAllPartitions();
```

Comment:
As I said before, this test should be moved to TestNonForwardQueryResultSystemScanner. (Maybe this class is also renamed, for example to TestInformationSchema). Here are some reasons. 
First, this class is for testing alter table operation rather than functionality of information schema. Second, in this implementation, several tests are mixed together, so it is difficult to figure out which lines test which functionality. Unit testing is to test the functionality of individual modules.